### PR TITLE
Tablegrid.defaultfilter

### PR DESCRIPTION
**Pull request by srabere**

### CODE REVIEW

File: tscommand-3271632f.tmp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+"C:\Servers\Lucca-UI\ts\deferred.cloak.ts" "C:\Servers\Lucca-UI\ts\filters-definitions.ts" "C:\Servers\Lucca-UI\ts\references.ts" --emitDecoratorMetadata --declaration --removeComments --experimentalDecorators --target ES5 --out .temp/lucca-ui-ts.js
```
Comment (lucienbertin) [inline tscommand-3271632f.tmp.txt]: this file shouldnt have been commited

**Comment by lucienbertin** [comment]
ne e1e test for this new feature

File: config/concurrent.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ module.exports = function(grunt, options){
 		options: {
 			logConcurrentOutput: true
 		},
-		debug: ['watch:js', 'watch:jsspec', 'watch:ts', 'watch:tsspec', 'watch:html', 'watch:sassdemo', 'karma:debug'],
+		debug: ['watch:js', 'watch:jsspec', 'watch:ts', 'watch:tsspec', 'watch:html', 'watch:sassdemo'],
```
Comment (PaulAndreLOUIS) [inline config/concurrent.js]: why ?

File: config/watch.js [hunk 3]
```diff
@@ -16,8 +16,8 @@ module.exports = function(grunt, options){
 		},
 
 		js: {
-			files: ['js/**/*.js'],
-			tasks: ['dist', 'karma:debug', 'jshint'],
+			files: ['js/**/*.js', 'demo/js/**/*.js'],
+			tasks: ['dist', 'jshint'],
 		},
```
Comment (PaulAndreLOUIS) [inline config/watch.js]: why ?

File: config/watch.js [hunk 4]
```diff
@@ -26,7 +26,7 @@ module.exports = function(grunt, options){
 
 		ts: {
 			files: ["ts/**/*.ts", "!ts/**/*.spec.ts", "!ts/**/*.e2e.ts"],
-			tasks: ["dist", "ts:test", "karma:debug", "tslint"]
+			tasks: ["dist", "ts:test", "tslint"]
 		},
```
Comment (PaulAndreLOUIS) [inline config/watch.js]: same here

Comment (lucienbertin) [inline config/watch.js]: dont touch my grunt config :angry:

File: demo/lucca-spe.html [hunk 5]
```diff
@@ -422,7 +424,7 @@ <h4 class="lui underline">Usage</h4>
 			rowspan: null,
 			textAlign: &quot;right&quot;,
 		},
-		children: [],
+		children: [],l
```
Comment (lucienbertin) [inline demo/lucca-spe.html]: ??

File: demo/lucca-spe.html [hunk 6]
```diff
@@ -422,7 +424,7 @@ <h4 class="lui underline">Usage</h4>
...
 	},
```
Comment (PaulAndreLOUIS) [inline demo/lucca-spe.html]: ?

**Comment by lucienbertin** [comment]
dont commit changes under the /dist directory

File: config/concurrent.js [hunk 7]
```diff
@@ -5,7 +5,7 @@ module.exports = function(grunt, options){
...
 		e2e: ['watch:e2esrc', 'watch:e2espec'],
```
Comment (lucienbertin) [inline config/concurrent.js]: grrr!!!!

File: tscommand-3271632f.tmp.txt (hunk 1 above)
Comment (PaulAndreLOUIS) [inline tscommand-3271632f.tmp.txt]: hum ?

**Pull request merged**